Assets/Live2D/Cubism/Framework/CubismAutoEyeBlinkInput.cs

```diff
@@ -194,6 +206,14 @@ private void UpdateEyeBlink()
                 }
                 case Phase.Idling:
                 {
+                    // Do not blink while playing non-loop motion
+                    if (DisableDuringAnimation && _animator is { } animator)
+                    {
+                        var stateInfo = animator.GetCurrentAnimatorStateInfo((int)CubismAnimationLayer.Motion);
+                        var stopAutoBlink = !stateInfo.loop && stateInfo.normalizedTime < 1;
+                        if (stopAutoBlink) return;
+                    }
+
```

Comment:
We cannot accept this change for the following reasons:
- It utilizes third-party namespaces and classes.
- The implementation differs from the Original Workflow specification.
- It depends on the `Mecanim` system.